### PR TITLE
HZN-132: JmxMonitor documentation

### PR DESCRIPTION
**Pull request by mfuhrmann**
https://issues.opennms.org/browse/HZN-132

### CODE REVIEW

File: opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/JmxMonitor.adoc [hunk 1]
```diff
@@ -0,0 +1,60 @@
+[[poller-jmx-monitor]]
+==== JmxMonitor
+
+The JMX monitor allows to test service availability of Java applications providing the Java Management extensions.
+The monitor offers the following functionalities:
+
+* test the connectivity to JMX application
```
Comment (mvrueden) [inline opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/JmxMonitor.adoc]: test the application's connectivity via JMX

File: opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/JmxMonitor.adoc [hunk 2]
```diff
@@ -0,0 +1,60 @@
+[[poller-jmx-monitor]]
+==== JmxMonitor
+
+The JMX monitor allows to test service availability of Java applications providing the Java Management extensions.
```
Comment (mvrueden) [inline opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/JmxMonitor.adoc]: I'd remove the `providing the Java Management extensions`

**Comment by mfuhrmann** [comment]
@mvrueden Which parameters are missing? I don't use any others in my configurations :-)

**Comment by mvrueden** [comment]
The following ones are missing: factory, protocol, rmi, urlPath, rmiServerPort, remoteJMX

See http://docs.opennms.org/opennms/branches/develop/guide-admin/guide-admin.html#_jmxcollector for more details. 
The defaults may vary for the jmx detector.

**Pull request merged**